sql/core/src/test/scala/org/apache/spark/sql/errors/QueryCompilationErrorsSuite.scala

```diff
@@ -1003,7 +1003,7 @@ class QueryCompilationErrorsSuite
         val exception = intercept[AnalysisException] {
           sql(queryWithTrailingComma)
         }
-        assert(exception.getErrorClass === "TRAILING_COMMA_IN_SELECT")
```

Comment:
The newly added logic comes from https://github.com/apache/spark/pull/48370/files#diff-1fd98ab15bfc9b8068b37cc5cfbc3376ee63ca2d18600a502db3012d24915424R1006